usaspending_api/references/tests/test_naics.py

```diff
@@ -0,0 +1,234 @@
+import json
+import pytest
+
+from model_mommy import mommy
+from usaspending_api.common.exceptions import UnprocessableEntityException
```

Comment:
Also this needs to go for Flake8.  After that I'm good to approve, but Tony will have to merge .